lib/utils/KalturaUtils.js

```diff
@@ -466,6 +466,19 @@ KalturaUtils = {
 		});
 
 		request.end();
+	},
+
```

Comment:
I have changed the places where null was relevant yet there are locations where we parse the configuration or numbers – so for now I keep the current behavior as to fail due to bad parsing